core/src/main/java/org/apache/iceberg/CatalogProperties.java

```diff
@@ -59,6 +59,9 @@ private CatalogProperties() {
   public static final String CLIENT_POOL_CACHE_EVICTION_INTERVAL_MS = "client.pool.cache.eviction-interval-ms";
   public static final long CLIENT_POOL_CACHE_EVICTION_INTERVAL_MS_DEFAULT = TimeUnit.MINUTES.toMillis(5);
 
+  public static final String FILTER_ICEBERG_TABLE = "filter.iceberg.table";
+  public static final boolean FILTER_ICEBERG_TABLE_DEFAULT = false;
```

Comment:
This is not a general catalog property because it is specific to Hive. Can you avoid modifying this class in core?

Reply:
ok, will move these properties to HiveCatalog.